docs/what-is-tizen/versions/tizen-3-0.md

```diff
@@ -1,42 +1,60 @@
-- **UHD support, Optimized Native Graphics**: Tizen 3.0 has also improved native graphics performance greatly. Tizen 3.0 supports UHD display format and the best part is it runs 30% faster compared to Tizen 2.4.
-- **Crosswalk**: Tizen now supports Crosswalk, an open-source based web runtime built with the latest releases of Chromium and Blink. It is designed to support multi-platform so developers can code once and reuse on other platforms. By using the Crosswalk Project, application developers can:
-- Use all the features available in modern web browsers: HTML5, CSS3, JavaScript.
-- Access the latest recommended and emerging web standards.
-- Use experimental APIs not available in mainstream web browsers.
-- **Latest Linux LTS (Long Term Support) Kernel **ensures stability, performance and enabling us to use the latest kernel features. Also it enables Tizen to be more secure and up to date with latest kernel features.
-- **64-bit support**: In order to extend the Tizen HW ecosystem to cover powerful CPU cores, Tizen 3.0 supports 64 bit CPUs including both Intel and ARM architectures. Utilizing high performance with more address space available in 64 bit CPUs, Tizen 3.0 will help developing high performance apps such as video editing and games.
-- **Multi-user support**: We understand that devices such as Smart TVs/Tablets/Cars are shared by all family members and each of us want to keep it personalized to our own preferences and needs. To accommodate such needs, Tizen 3.0 supports multiple users via logins and multiple sessions. Using Tizen 3.0, each user of a device can control his or her own services, private contents and favorite settings independently from other users.
-- **Simple and Flexible Security**: Security has been one of Tizen’s major strengths. Strong security, however, has sometimes caused difficulties in managing security policies. To solve these difficulties, Tizen 3.0 introduces a new security model. By grouping same privileged system resources, the complexity of policy management at the kernel level is greatly reduced. For user space access control, we bring in a policy checker called Cynara that checks the application has the privilege to use privileged APIs. This provides a way to implement flexible, fast and safe policy-checking service for granting applications any necessary access to system resources, while it allows access control as API level granularity. Tizen 3.0 also introduces a new anti-virus framework and privacy guard. The privacy guard reports statics on how applications are using each resource such as GPS, Connectivity, and users can control the configuration of the same.
```

Comment:
“the user can control the configuration of the same”
Same of what?

Reply:
The original doc is https://developer.tizen.org/tizen/tizen/tizen-3.0.
I think our team requested for you to review all docs before publishing, but  I'm not sure LB reviewed it when it was published.

I think the sentence is award.
*The privacy guard reports statics on how applications are using each resource such as GPS, Connectivity, and users can control the configuration of the same.*

You can change the sentence.
How about this?
*The privacy guard reports statistics on how applications are using each resource such as Sensors,  GPS, and Connectivity, and users can control the configuration of the privacy on the Settings application.*

Reply:
I would prefer this:
"The privacy guard reports statistics on how applications are using each resource, such as sensors, GPS, and Bluetooth, and the user can control their privacy configuration in the Settings application."